test/fail_compilation/diag13609b.d

```diff
@@ -1,8 +1,9 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/diag13609b.d(9): Error: { } expected following struct declaration
-fail_compilation/diag13609b.d(9): Error: declaration expected, not ':'
+fail_compilation/diag13609b.d(11): Error: basic type expected, not EOF
```

Comment:
It's not a regression. It's a consequence of changing how the parsing is done to treat templates/union/struct/class the same as far as parsing goes, and sort out errors later. The file contains ':' followed by end of file. The parser sees the : and looks for base classes, but finds end of file. Hence the message.